tests/test_speech_tools.py

```diff
@@ -52,3 +52,15 @@ def test_indescribable(self) -> None:
         description = get_additive_description(Indescribable())
 
         assert description == "something indescribable"
+
+
+class TestRepresentProp:
+    def test_str(self):
+        val = "hello\nworld!"
+        
+        assert represent_prop(val) == "'hello\\nworld!'"
+
+    def test_int(self):
+        val = 1234
+
+        assert represent_prop(val) == "<1234>"
```

Comment:
If we wanted to, we could parametrize this as `test_everything_else` and verify that a bunch of different classes get wrapped in the `<>`s.

Reply:
You're not wrong...

Separating them gives you a clear indication of where a bug lies. If somehow we only break how ints are displayed, when we run the tests `TestRepresentProp:test_int` will inform us.  While in this case that's not a significant difference in how we would go about fixing it, it is often helpful to have that separation to quickly diagnose a bug.  

Code wise it's a couple extra lines. /shrug

Reply:
Parametrized tests still show what the values are, so we'd have that same level of granular feedback. We can also give each parameter'd test a specific name if we'd like to.